test/lib/buildFactory.test.js

```diff
@@ -116,12 +117,16 @@ describe('Build Factory', () => {
         const eventId = '12345f642bbfd1886623964b4cff12db59869e5d';
         const sha = 'ccc49349d3cffbd12ea9e3d41521480b4aa5de5f';
         const scmUri = 'github.com:12345:master';
+        const scmRepo = {
+            name: 'screwdriver-cd/models'
+        };
+        const prRef = 'pull/3/merge';
         const username = 'i_made_the_request';
         const dateNow = Date.now();
         const isoTime = (new Date(dateNow)).toISOString();
         const container = 'node:4';
         const steps = [
-            { name: 'sd-setup' },
+            { command: 'git clone', name: 'checkout-code' },
```

Comment:
If this step is named something other than `sd-setup`, then it needs to start with `sd-` to keep from stepping on user configs. 

If the launcher is still doing other special commands in `sd-setup` maybe those should also be moved out of launcher and inserted by the models. Maybe that should be done in another PR. We want to keep the `sd-` steps to a minimum.